README.md

```diff
@@ -116,7 +116,7 @@ api.listJobs().then(function(metadata) {
 });
 ```
 
-Create a job request:
+Create a job request to run on a data. Pass certain `Analytic-Name(Type)` as `<analytic>`, particular `Data Id` as `dataId`, give data type `video` or `images` as `<input>`,and certain paramter and it's value based on analytic type you choose to run. For more information on `input` and `parameters` refers to `analytic.json`   :
```

Comment:
"Create a job request to perform an analytic on a data, where `<analytic>` is the analytic's name, `dataId` is the desired data file's unique ID, and any `<param#>` value are any of the permitted parameters based on the analytic's documentation."

Reply:
ok. thanks. I will modify the text